gherkin/c/testdata/good/tags.feature.pickles.ndjson

```diff
@@ -3,29 +3,29 @@
     "astNodeIds": [
       "4"
     ],
-    "id": "24",
+    "id": "34",
```

Comment:
@aslakhellesoy, @vincent-psarga  I don't understand why these ids changed. These scenarios didn't change position. Is there an implicit ordering going on?